keras/engine/topology.py

```diff
@@ -3027,6 +3027,27 @@ def preprocess_weights_for_loading(layer, weights,
             weights[0] = np.transpose(weights[0], (3, 2, 0, 1))
             if layer.__class__.__name__ == 'ConvLSTM2D':
                 weights[1] = np.transpose(weights[1], (3, 2, 0, 1))
+
+    # convert the weights of CuDNNLSTM so that they could be loaded into LSTM
+    if layer.__class__.__name__ == 'LSTM':
+        # determine if we're loading a CuDNNLSTM layer from the number of bias weights:
+        # CuDNNLSTM has (units * 8) weights; while LSTM has (units * 4)
+        units = weights[1].shape[0]
+        bias = weights[2]
+        if len(bias) == units * 8:
```

Comment:
Please put this paragraph in the `if layer.__class__.__name__ == 'LSTM':` that already exists above

Reply:
I thought the `if layer.__class__.__name__ == 'LSTM':` that already exists above will only be executed if `original_keras_version == '1'`. This conversion is for weights saved under Keras>=2.0.8, isn't it?

Reply:
You're right. Never mind then.